modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -357,34 +357,40 @@
 	<tag>
 		<description></description>
 		<name>checkbox</name>
-		<tag-class>com.liferay.frontend.taglib.clay.servlet.taglib.soy.CheckboxTag</tag-class>
-		<body-content>empty</body-content>
+		<tag-class>com.liferay.frontend.taglib.clay.servlet.taglib.CheckboxTag</tag-class>
+		<body-content>JSP</body-content>
```

Comment:
why this change?

Reply:
Multiple tags have `JPS` instead of `empty` set there, for example button: https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld#L198. So I changed it after noticing that

Reply:
AFAIK `body-content` defines the type of body the tag accepts.

So `body-content` should be `JSP` if you accept html, other tags, etc as body content as in:
```
<clay:checkbox attrs...>
   Body content
</clay: checkbox>
```

If you don"t expect any body content (which I think is the case here? You are always returning `SKIP_BODY` in `processStartTag`) you need to se it as `empty`

```
<clay:checkbox attrs... />
```

Reply:
Oh I see, that's what that stands for, thanks! It actually makes sense.